webapp/src/ts/modules/reports/reports.component.ts

```diff
@@ -193,6 +194,15 @@ export class ReportsComponent implements OnInit, AfterViewInit, OnDestroy {
       report.lineage = report.subject && report.subject.lineage || report.lineage;
       report.unread = !report.read;
 
+      // remove the lineage level that belongs to the offline logged-in user
+      if (userLineageLevel && report?.lineage?.length) {
+        report.lineage = report.lineage.filter(level => level);
+        const item = report.lineage?.length && report.lineage[report.lineage.length -1];
```

Comment:
```suggestion
        const item = report.lineage[report.lineage.length -1];
```

Minor, but don't need to check the lineage length here again.